src/display/canvas.js

```diff
@@ -604,8 +604,15 @@ class CanvasExtraState {
     this.maxY = Math.max(this.maxY, y);
   }
 
-  updateCurvePathMinMax(transform, x0, y0, x1, y1, x2, y2, x3, y3) {
+  updateCurvePathMinMax(transform, x0, y0, x1, y1, x2, y2, x3, y3, minMax) {
     const box = Util.bezierBoundingBox(x0, y0, x1, y1, x2, y2, x3, y3);
+    if (minMax) {
+      minMax[0] = Math.min(minMax[0], box[0], box[2]);
+      minMax[1] = Math.min(minMax[1], box[0], box[2]);
+      minMax[2] = Math.min(minMax[2], box[1], box[3]);
+      minMax[3] = Math.min(minMax[3], box[1], box[3]);
```

Comment:
Are all of these lines actually *supposed* to use `Math.min(...)`, or was there a bit too much copy-pasting happening here?